operator/internal/manifests/options.go

```diff
@@ -126,6 +127,8 @@ type TLSProfileSpec struct {
 	MinTLSVersion string
 }
 
+var ErrInvalidPerTenantConfig error = errors.New("invalid per-tenant config")
```

Comment:
While testing I read the error message and it would not be obvious what was the issue 
```suggestion
var ErrPerTenantConfigInvalid error = errors.New("per-tenant config cannot contain empty keys")
```